AdyenDropIn/Components/PaymentMethodListComponent.swift

```diff
@@ -46,30 +46,33 @@ internal final class PaymentMethodListComponent: ComponentLoader, PresentableCom
     private let brandProtectedComponents: Set = ["applepay"]
     
     internal lazy var listViewController: ListViewController = {
-        func item(for component: PaymentComponent) -> ListItem {
-            let showsDisclosureIndicator = (component as? PresentableComponent)?.requiresModalPresentation == true
-            
-            let displayInformation = component.paymentMethod.localizedDisplayInformation(using: localizationParameters)
-            let isProtected = brandProtectedComponents.contains(component.paymentMethod.type)
+        func item(for paymentMethod: PaymentMethod) -> ListItem {
+            let displayInformation = paymentMethod.localizedDisplayInformation(using: localizationParameters)
+            let isProtected = brandProtectedComponents.contains(paymentMethod.type)
             let listItem = ListItem(title: displayInformation.title,
                                     style: style.listItem,
                                     canModifyIcon: !isProtected)
             listItem.identifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: listItem.title)
-            listItem.imageURL = LogoURLProvider.logoURL(for: component.paymentMethod, environment: apiContext.environment)
+            listItem.imageURL = LogoURLProvider.logoURL(for: paymentMethod, environment: apiContext.environment)
             listItem.trailingText = displayInformation.disclosureText
             listItem.subtitle = displayInformation.subtitle
-            listItem.showsDisclosureIndicator = showsDisclosureIndicator
-            listItem.selectionHandler = { [unowned self, unowned component] in
-                guard !(component is AlreadyPaidPaymentComponent) else { return }
-                self.delegate?.didSelect(component, in: self)
+
+            // Unsure how to handle this
+//            let showsDisclosureIndicator = (component as? PresentableComponent)?.requiresModalPresentation == true
```

Comment:
This shows `>` chevron on list.. we can probably omite this

Reply:
I'm not sure if we need that. To me it seems unnecessary, since the disclosure indicator usually means using native navigation, and something sliding from the right. Maybe removing it is okay

Reply:
👍